.circleci/config.yml

```diff
@@ -9,27 +9,42 @@ commands:
           keys:
             - composer-v1-{{ checksum "composer.json" }}
             - composer-v1-
-      - run: composer install -n --prefer-dist
-      - persist_to_workspace:
-          root: .
-          paths:
-            - composer.*
-            - .snyk
+      - run: composer update -n --prefer-dist
       - save_cache:
           key: composer-v1-{{ checksum "composer.json" }}
           paths:
             - vendor
-
-jobs: 
-  php_7: 
-    docker: 
+  test:
+    steps:
+      - run: ./vendor/bin/phpunit
+jobs:
+  php_71:
+    docker:
       - image: circleci/php:7.1
     steps:
       - prepare
+      - test
+  php_72:
+    docker:
+      - image: circleci/php:7.2
+    steps:
+      - prepare
+      - test
+  php_73:
+    docker:
+      - image: circleci/php:7.3
+    steps:
+      - prepare
+      - test
   snyk:
     docker:
       - image: snyk/snyk-cli:composer
     steps:
+      - persist_to_workspace:
```

Comment:
Let me know if I broke something here 🙈. I don't have a snyk ENV

Reply:
Hehe. Yes, these 2 files should be available from the previous job in order for snyk job to execute.